sdk/identity/Azure.Identity/tests/DefaultAzureCredentialTests.cs

```diff
@@ -325,16 +340,18 @@ public void DisableMetadataDiscoveryOptionIsHonoredWhenTrue(Type availableCreden
             {
                 Assert.Ignore($"Credential {availableCredential.Name} does not support disabling instance discovery");
             }
-            DefaultAzureCredentialOptions options = GetDacOptions(availableCredential, true);
 
             using (new TestEnvVar(new Dictionary<string, string> {
                     { "AZURE_CLIENT_ID", "mockclientid" },
                     { "AZURE_CLIENT_SECRET", null},
                     { "AZURE_TENANT_ID", "mocktenantid" },
                     {"AZURE_USERNAME", "mockusername" },
                     { "AZURE_PASSWORD", "mockpassword" },
-                    { "AZURE_CLIENT_CERTIFICATE_PATH", null } }))
+                    { "AZURE_CLIENT_CERTIFICATE_PATH", null },
+                    { "AZURE_FEDERATED_TOKEN_FILE", "c:/temp/token" } }))
```

Comment:
Is this hard coded path always valid cross platform?

Reply:
no, I can change it. But the value doesn't really matter, in this case. it just needs to be set and not null for the test to function.